.travis.yml

```diff
@@ -10,14 +10,25 @@ script:
 before_deploy: scripts/before-deploy.sh
 
 deploy:
-  provider: pages
-  skip_cleanup: true
-  github_token: $GITHUB_TOKEN
-  keep_history: true
-  local_dir: dist
-  repo: $DEPLOY_REPO
-  target_branch: master
-  name: $DEPLOY_NAME
-  email: $DEPLOY_EMAIL
-  on:
-    branch: feature/layout-superhero
+  - provider: pages
+    skip_cleanup: true
+    github_token: $GITHUB_TOKEN
+    keep_history: true
+    local_dir: dist
+    repo: $DEPLOY_REPO
+    target_branch: master
+    name: $DEPLOY_NAME
+    email: $DEPLOY_EMAIL
+    on:
+      branch: superhero-production
```

Comment:
The only problem is branch naming. Should we merge `feature/layout-superhero` to `master`? This line proposes to use `superhero-production` as the temporary master branch.

Reply:
We can keep it `superhero-production` for now as you suggested and merge in `master` after we have agreement.